.gitignore

```diff
@@ -3,3 +3,4 @@ dist/*
 .vscode
 .DS_STORE
 /yarn-error.log
+yarn.lock
```

Comment:
Curious what exactly is wrong with this yarn.lock? If we do a `yarn upgrade @folio/ui-eholdings` or `yarn upgrade-interactive @folio/ui-eholdings` shouldn't it minimally shift all dependencies just enough so that the version constraints are satified?

Reply:
The problem is that this repo's `stripes-core`, `stripes-*`, etc. gets out of sync with the dependencies we're testing `ui-eholdings` against. The `stripes` ecosystem's versioning can't be trusted while we're creating version tags on every PR, so the most stable way to build the "platform" in this repo is by using the exact same `yarn.lock` we use in `ui-eholdings`. That integration strategy won't work for platforms meant to include more than just one UI module.